config/canonical.json

```diff
@@ -16791,6 +16793,8 @@
     "match": ["shop/bakery|BackWerk"],
     "tags": {
       "brand": "Backwerk",
```

Comment:
The W should be capitalized throughout this entry

Reply:
Hhhmmm, normally its good to go with established tagging and not create new tags, but in the case of the brand tag there is only 3 of them mapped in lowercase anyway. So I guess it doesn't matter. With the name tag there's 179 in lower case and 56 with a capital letter. So its probably better to keep that one lowercase.